README.md

```diff
@@ -1,7 +1,13 @@
 # PuppetDB
 
-[docs]: https://docs.puppetlabs.com/puppetdb/latest
+[docs]: https://docs.puppet.com/puppetdb/latest
 
 PuppetDB is the fast, scalable, and reliable data warehouse for Puppet. It caches data generated by Puppet, and gives you advanced features at awesome speed with a powerful API.
 
 For documentation on this product, consult the [latest documentation][docs].
+
+## Maintenance
+
+Maintainers: Ryan Senior <ryan.senior@puppet.com>, Wyatt Alt <wyatt@puppet.com>, Rob Browning <rlb@puppet.com>, AJ Roetker <andrew.roetker@puppet.com>, Russell Mull <russell.mull@puppet.com>, Ken Barber <ken@puppet.com>
```

Comment:
@lutter This doesn't seem like the right thing for a Puppet product vs. a shared library (public or internal). In product terms, people working on it is more fluid. Is puppet-users or some other generic public forum a better address for this?